<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ffffff</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="NomeInicial">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>301</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;
font-weight: bold;
color: #000000</string>
    </property>
    <property name="text">
     <string>            BEM VINDO!</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>230</y>
      <width>291</width>
      <height>189</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="txt_nome">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    font-size: 14px;
    color: black;
    border: 1px solid gray;
    border-radius: 5px;
    padding: 5px;
}
QLineEdit:placeholder {
    color: gray;
    font-style: italic;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Usu√°rio</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txt_senha">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    font-size: 14px;
    color: black;
    border: 1px solid gray;
    border-radius: 5px;
    padding: 5px;
}
QLineEdit:placeholder {
    color: gray;
    font-style: italic;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Senha</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="BotaoDeEntrada">
       <property name="styleSheet">
        <string notr="true">background-color: #E8E8E8;
border-radius: 5px;
padding: 5px;
color: #000000</string>
       </property>
       <property name="text">
        <string>ENTRAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #B88AD1;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1302</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuinical">
    <property name="title">
     <string>inical</string>
    </property>
   </widget>
   <addaction name="menuinical"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
